mzLib/UsefulProteomicsDatabases/ProteinDbLoader.cs

```diff
@@ -260,25 +306,18 @@ public static List<Protein> LoadProteinDb(string proteinDbLocation, bool onTheFl
                         if ((fasta.Peek() == '>' || fasta.Peek() == -1) && accession != null && sb != null)
                         {
                             var sequence = sb.ToString();
-                            var protein = new Protein(sequence, accession, oneBasedModifications, oneBasedBeginPositions.ToArray(), oneBasedEndPositions.ToArray(), peptideTypes.ToArray(), name, full_name, offset, false, IsContaminant);
+                            var protein = new Protein(sequence, accession, null, null, null, null, name, full_name, offset, false, IsContaminant, null);
```

Comment:
Hmm, those four nulls after "accession" should probably be switched back, right?